test/README.md

```diff
@@ -1,72 +1,38 @@
-    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-    "version": "0.2.0",
-    "configurations": [
-        {
-            "name": "Transpile & Run",
-            "preLaunchTask": "transpile",
-            "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/ava",
-            "runtimeArgs": [
-                "debug",
-                "--break",
-                "--verbose",
-                "--color",
-                "--serial",
-                "--watch=false",
-                "build/ts-to-es6/test/**/${fileBasenameNoExtension}.js",
-            ],
-            "request": "launch",
-            "skipFiles": [
-                "<node_internals>/**"
-            ],
-            "type": "node",
-            "outputCapture": "std",
-            "port": 9229,
-            "sourceMaps": true,
-            "trace": true,
-            "protocol": "auto",
-            "console": "integratedTerminal"
-        },
-        {
-            "name": "Run",
-            "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/ava",
-            "runtimeArgs": [
-                "debug",
-                "--break",
-                "--verbose",
-                "--color",
-                "--watch=false",
-                "--fail-fast=false",
-                "build/ts-to-es6/test/**/${fileBasenameNoExtension}.js",
-            ],
-            "request": "launch",
-            "skipFiles": [
-                "<node_internals>/**"
-            ],
-            "type": "node",
-            "outputCapture": "std",
-            "port": 9229,
-            "sourceMaps": true,
-            "trace": true,
-            "protocol": "auto",
-            "console": "integratedTerminal"
-        },
-    ]
-}
-```
+0. Ensure you can run a specific test noted above first
+1. Go to the test file you want to debug
+2. Highlight a specific test, including the quotes
+3. Do one of the following to start debugging
+   - MacBook Touch Bar - Tap play button 
+   - Press F5
+   - "Run" > "Start debugging"
+   - Open "Run and Debug" on the left bar (or Control + Shift + D)
+      - Select  "Debug AVA by selected test (w/ quotes)"
 
-2. Go to the test file and tap the play button (MacBook Touch Bar) or go to **Run > Start debugging**
\ No newline at end of file
+![image](https://user-images.githubusercontent.com/645861/111061933-ada31000-845a-11eb-86c9-851ef378c592.png)
+
+If you need to configure any advanced parameters in the test edit `.vscode/launch.json`
+
+## Testing from CLI
+1. Run all and watch file changes: `yarn test`
+2. Run all with no watch: `yarn test:noWatch`
+3. Run specific file: `<one of above commands> <path to file>`
+   - e.g: `yarn test:noWatch test/unit/context/sw/ServiceWorker.ts`
+3. Run specific test in a file: `<one of above commands> <path to file> -m "name of test here"`
+   - e.g: `yarn test:noWatch test/unit/context/sw/ServiceWorker.ts -m "onPushReceived - Ensure undefined payload does not show"`
```

Comment:
I got the following when running the test:
```
Debugger attached.

  ✖ Provide the path to the test file you wish to debug
Waiting for the debugger to disconnect...
```

Let's add a troubleshooting section for the solution to this as others may get the same in the future

Reply:
> I got the following when running the test:

I didn't have any issue with a debugger not attaching when running a test from the terminal inside of VScode. However after testing I noticed it takes about 2x longer to start so I would recommend debugging from VSCode noted above.

> Let's add a troubleshooting section for the solution to this as others may get the same in the future

Not sure what to add here, not sure why that didn't work for but since it isn't the recommend way to debug I don't see it as a blocker.